docs/src/main/asciidoc/application-configuration-guide.adoc

```diff
@@ -257,6 +257,16 @@ quarkus.http.port=9090
 
 And then set the `QUARKUS_PROFILE` environment variable to 'staging' to activate my profile.
 
+==== Miscellaneous
+The default Quarkus application runtime profile is set to the profile used to build the application.
+For example:
+[source,shell]
+----
+mvn package -Pnative -Dquarkus-profile=prod-aws`
```

Comment:
Note: we need to update this as soon as https://github.com/quarkusio/quarkus/pull/3159 is in.